src/test/java/io/openliberty/tools/intellij/it/SingleModJakartaLSTestCommon.java

```diff
@@ -12,6 +12,7 @@
 import com.automation.remarks.junit5.Video;
 import com.intellij.remoterobot.RemoteRobot;
 import com.intellij.remoterobot.fixtures.JTreeFixture;
+import io.openliberty.tools.intellij.it.Utils.ItConstants;
```

Comment:
I'd like to suggest using `import static io.openliberty.tools.intellij.it.Utils.ItConstants.*` so that you do not have to specify the class name every time you use a constant in all these test files. It gets repetitive.